java/database/LoginService.java

```diff
@@ -9,83 +9,105 @@
 
 import entities.Login;
 
-public class LoginService extends Service {
+public class LoginService {
 	
 		
 static Connection con;
 
 
-	public LoginService(Connection conn) {
-		super(conn);
-	}
+public LoginService(Connection con) {
+	super();
+	this.con=con;
+}
 
+private int getPK() {
```

Comment:
This is the old "getPK()"  this should be changed to the one that takes an integer input.  Check in one of the other services for the updated version.